src/tests/sqlsmith/src/sql_gen/expr.rs

```diff
@@ -90,7 +91,11 @@ impl<R: Rng> SqlGenerator<'_, R> {
         // - `a1 >= a2 IN b`
         // ...
         // We just nest compound expressions to avoid this.
-        let range = if context.can_gen_agg() { 100 } else { 50 };
+        let range = if context.can_gen_agg(self.should_generate(Feature::Agg)) {
```

Comment:
Seems more readable to just flatten it.
```suggestion
        let range = if context.is_inside_agg && self.should_generate(Feature::Agg) {
```

Reply:
Remove `can_gen_agg() ` from SqlGeneratorConext now. Thanks!